app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,41 @@
   </head>
 
   <body>
+  <!-- navbar section - css from _navbar_car.scss in components -->
```

Comment:
is this html in the application.html.erb? I think it should be in it's own folder and then rendered.